### PR TITLE
Update PortScanner.py

### PR DESCRIPTION
**Pull request by imvarungupta**
Adding Extra line

### CODE REVIEW

File: PortScanner.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import optparse
 import socket
+
```
Comment (ShubhamRajvanshi10) [inline PortScanner.py:3]: Why extra line?

Comment (imvarungupta) [reply PortScanner.py:3]: import block should be separated from the code

Comment (ShubhamRajvanshi10) [reply PortScanner.py:3]: Aaeh may be...not interested at this time ...so cancelling your request only for fun.. :-)

**Comment by ShubhamRajvanshi10** [comment]
Sorry buddy.. :-)

**Pull request closed without merging**